src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2650,8 +2650,7 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
             $messageToSerializersMapping[$message] = array_keys($messageToSerializersMapping[$message]);
         }
 
-        $container->getDefinition('messenger.signing_serializer')
-            ->replaceArgument(2, $messageToSerializersMapping);
+        $container->setParameter('messenger.message_to_serializers_mapping', $messageToSerializersMapping);
```

Comment:
why add this parameter? It's not needed for the fix isn't it?
better remove it IMHO